torch/onnx/_internal/fx/export.py

```diff
@@ -415,6 +433,18 @@ def _export_fx_to_onnxscript(fx_module_with_metadata, opset_version):
             assert isinstance(output, (graph_building.TorchScriptTensor, tuple)), type(
                 output
             )
+            # op-level validation
+            # TODO(titaiwang): Change ORTEvaluator to ReferenceEvaluator
+            # Symbolic_fn should have the same output as node.target (torch ops)
+            with evaluator.default_as(evaluator.ORTEvaluator()):
+                expected_output = node.target(*torch_args, **torch_kwargs)
+                numpy_args = [
+                    arg.numpy() if isinstance(arg, torch.Tensor) else arg
+                    for arg in torch_args
+                ]
+                ort_output = symbolic_fn(*numpy_args, **torch_kwargs)
+                torch.testing.assert_close(expected_output.numpy(), ort_output)
```

Comment:
Handle multiple output as well? This block of code can be extracted into a function for clarity. (439-446)

Reply:
Also an option to disable the check would be handy too.

Reply:
Another PR to integrate this https://github.com/pytorch/pytorch/pull/94084